### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by danutibe07**
Use React components.
Use React props.
Use React Router.
Connect React and Redux.
Handle events in a React app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,18 +59,6 @@ We select an API that provides numeric data about a topic that we like and then
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🚀 Live Demo <a name="live-demo"></a>
-
-<!-- - [You can check the live demo here](https://643d6e77f0f1ad007d2a1498--idyllic-semifreddo-83ba58.netlify.app/) -->
```
Comment (prolajumokeoni) [inline README.md]: - [ ] Kindly deploy your application and make sure that the link is visible on your readme, please also ensure that the home and detail page works as expected 👍🏿

File: README.md [hunk 2]
```diff
@@ -61,18 +59,6 @@ We select an API that provides numeric data about a topic that we like and then
...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-## 📺 Presentation <a name="presentation"></a>
-
-<!-- - [ScreenRec Presentation](https://screenrec.com/share/dl4csmS2Vn) -->
```
Comment (prolajumokeoni) [inline README.md]: - [ ] Kindly document your project by recording a video following all the project [documentation requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#project-documentation), this presentation is currently missing in your project and 
- [ ] Please make sure that the link works properly and is visible   on your readme👍🏿 

Screenshot_17

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @danutibe07 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Created `Home` page and  `Detail` page 👏🏿  
✔️  Crypto can be filtered by name 💪🏿 



## Required Changes ♻️
Behance Design          |  Your design     
:-------------------------:|:-------------------------:
99235e31579789 56617c411f336  | Screenshot_18

- [ ] Kindly make your mobile design look more like the Behance layout by making two cards to be on a row, adding the alternating card color layout 🏁  and other card layout features like aligning your image and text , implementing these features will make your design look very close to the Behance design, and  Please use the Typography stated in the Behance design, please don't forget you can use any color you want 👍🏿  
- [ ] Please add unit tests for major components of  your app and add test for some pure functions in Redux code, you can perform a mocking data fetching and Redux Store connection
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @danutibe07 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Highlights 🎯 
✔️ Descriptive PR
✔️ Used GitFlow
✔️ No linter errors

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Well done for recording the presentation video just like the previous reviewer suggested. However, there is only one thing missing: The video should contain a highlight about a piece of the code the student finds interesting or is proud of making. You and re-do the video or record that part and append it with the use of any capable software 👍 
- [ ] Just like the previous reviewer suggested, please use the specified Typography in the behance design as shown below 👇 
image


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @danutibe07 :wave: 

Well done!!! 👍 
You still need to fix some issues before moving to the next project. :running_man: 

## Highlights 🌟

Alt Text

✔️ Linters checks are passing.
✔️ You have used the correct git-flow.
✔️ Readme is professional.

## Required Changes ♻️

- [ ] A professional presentation can help us to attract the interviewer or the project reviewer in our future jobs. In order to create a professional presentation we need to take care of the following things:
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description of the project in professional English. 
- The topic and purpose of the project are made clear.
- The video contains a demo of the project, highlighting all the project features.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

> Currently in the video I only can see the homepage of your website, nothing else. You haven't presented any functionality or haven't presented any features, Such as:
> - After clicking any content which pages it is redirecting.
> - After searching what kind of result it is showing.
> - A piece of code from your codebase you are proud of or found interesting.

> ***Kindly record the video again with all the suggested changes.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
# STATUS: CAPSTONE APPROVED 💯 

Hi @danutibe07 , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Professional video presentation, less than 5 minutes
:heavy_check_mark: All pages working as required
:heavy_check_mark:  Tests added and all passing
:heavy_check_mark: Linter checks passing

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**